code-examples/missingNumber/src/MissingNumberTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
```

Comment:
This was weird: I had to use JUnit5.3. Even though I imported JUnit4 the test methods would not run unless I used JUnit5.3... hmmmm..

Reply:
Still have issues with this. Need to show you when I see you I think!